tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -439,8 +480,8 @@ def test_snapshot_create_full_latency(
     # - Rootfs: Ubuntu 18.04
     # - Microvm: 2vCPU with 256/512 MB RAM
     # TODO: Multiple microvm sizes must be tested in the async pipeline.
-    microvm_artifacts = ArtifactSet(artifacts.microvms(keyword="2vcpu_512mb"))
-    microvm_artifacts.insert(artifacts.microvms(keyword="2vcpu_256mb"))
+    microvm_artifacts = ArtifactSet(artifacts.microvms(keyword="2vcpu_256mb"))
```

Comment:
Just curious about the reorder? Is the 256mb test quicker so if it fails we get feedback faster?

Reply:
No; It made it easier to update the values inside the LOAD_LATENCY_BASELINES hashmap.